src/main/resources/db/migration/local+dev/20220913131809__mock_characteristics.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO characteristic (id, name, service_scope)
+VALUES ('952790c0-21d7-4fd6-a7e1-9018f08d8bb0','Park nearby','CAS3');
```

Comment:
Again, not really an issue but I would also consider generating the UUIDs rather than just changing the last few characters so that they're easier to tell apart when debugging.  There's a good plugin for IntelliJ named "UUID Generator" by Leo Million that adds an option to the `Alt+Insert` menu to generate a UUID.  This works with multi-line selections so if you go

```
INSERT INTO characteristic (id, name, service_scope) VALUES
   ('', 'a', 'CAS3'),
   ('', 'b', 'CAS3'),
   ('', c', 'CAS3')
```

Then hold down alt and drag down the empty IDs you can generate them all in one go